src/main/webapp/app/pages/userPage/UserPage.tsx

```diff
@@ -340,73 +425,7 @@ export default class UserPage extends React.Component<IUserPage> {
         country: values.country,
       };
       this.getUserStatus = PromiseStatus.pending;
-      await this.verifyUserEmail();
```

Comment:
We can remove `async` from function now that this call is removed

Reply:
Nevermind this, was fixed in recent commit

Reply:
Yup, just fixed actually. Thanks for catching this.